src/DFGNode.cpp

```diff
@@ -295,19 +391,47 @@ string DFGNode::getOpcodeName() {
       Function *func = ((CallInst*)m_inst)->getCalledFunction();
       if (func) {
         string newName = func->getName().str();
-	string removingPattern = "llvm.vector.";
-	int pos = newName.find(removingPattern);
-	if (pos == -1)
-	  pos = newName.find("llvm.");
-	newName.erase(pos, removingPattern.length());
+        string removingPattern = "llvm.vector.";
```

Comment:
Ah, I got how opcodename is different... We may need another field/variable `fusedOpCodeName` to distinguish between them? WDYT?

Reply:
I agree with that...Current implementation to handle this issue is a little confusing.